docs/team.rst

```diff
@@ -0,0 +1,128 @@
+Read the Docs Team
+==================
+
+readthedocs.org is the largest open source documentation hosting service.
```

Comment:
I've been using "The Read the Docs community site" in place of readthedocs.org. Maybe there is something to settle on here eventually, I never know what to write.